common/core/desktop/include/keyman/keyboardprocessor_debug.h

```diff
@@ -25,6 +25,21 @@
  */
 #define DEBUG_MAX_CONTEXT 80
 
+/**
+ * The number of stores that can be processed in a rule. This is taken from
+ * MAXSTOREOFFSETS in keyman32 (Windows) and is purely a convenience value.
+ * We can increase it if there is a demonstrated need.
+ */
+#define DEBUG_MAX_STORE_OFFSETS 20
```

Comment:
Not to be too picky, but perhaps `DEBUG_MAX_RULE_STORE_OFFSETS` may be slightly better?  I've definitely seen keyboards with more total stores, which is what the current name suggests.  Reading the comment corrects the notion, but still.